packages/react-ui-components/src/SelectBox_Option_MultiLineWithThumbnail/style.css

```diff
@@ -1,8 +1,25 @@
 .multiLineWithThumbnail__item {
+    display: grid;
+    grid-template-columns: auto 1fr;
+    gap: -var(--spacing-Half);
+    align-items: start;
     box-sizing: content-box;
     background-color: var(--colors-ContrastDarkest);
 }
 
+.multiLineWithThumbnail__item span {
```

Comment:
A looong tittle makes the whole element wider and the ellipsis of the secondary and tertiary labels disappear.